### DISCUSSION TITLE
Post processing variable values

### DISCUSSION THREAD

**Discussion by vermaprk** [idaholab/moose#24150] [Q&A Modules: Phase field] [answered]
Hi
I am doing phase field modelling with eta (variable) as the order parameter. eta=1 for phase1 and eta = 0 for phase 2 and 0<eta<1 for the diffused interface between the two phases. I want to track the interface at the end of simulation with a condition as:

For 0.4<eta<0.6 I want all the x, y coordinates. I have no experience with handling exodus file system. So can I generate this data using the moose postprocessor.

The post processed file should look like:
x     y      eta

Thanks

**Comment by GiudGiud** [comment]
Hello

Is this just for output? You can use thresholds in paraview to limit the elements shown to those meeting a certain value. 

Guillaume

**Comment by GiudGiud** [comment, marked as answer]
This object will output all the values in each element. 
https://mooseframework.inl.gov/source/vectorpostprocessors/ElementValueSampler.html

What you COULD do to get just eta between two values is use this user object (below) to modify the subdomains
https://mooseframework.inl.gov/source/userobjects/CoupledVarThresholdElementSubdomainModifier.html
then use the ElementVlaueSampler on the new subdomain using the block parameter

BUT I reckon it's easier to just postprocessor the CSV for values of eta you care about
